code/index.html

```diff
@@ -3,28 +3,347 @@
+        <input 
+        id="oneTwo" 
+        type="radio"
+        required
+        value="one_two"
+        name="groupSizeValue"
+        class="form-input-radio">
+        
+        <span class="form-custom-radio"></span>
+
+        <label 
+        for="oneTwo" 
+        class="form-label-input">
+        1 - 2</label>
+        </div>
+            
+      <div class="form-size-group">
+        <input 
+        id="twoFour" 
+        type="radio"
+        required
+        value="two_four"
+        name="groupSizeValue"
+        class="form-input-radio">
+
+        <label 
+        for="twoFour"
+        class="form-label-input">
+        2 - 4</label>
+      </div>
+
+      <div class="form-size-group">
+        <input 
+        id="fiveTen" 
+        type="radio"
+        required
+        value="five_ten"
+        name="groupSizeValue"
+        class="form-input-radio">
+
+        <label 
+        for="fiveTen"
+        class="form-label-input">
+        5 - 10</label>
+      </div>
+          
+      <div class="form-size-group">
+        <input 
+        id="tenTwenty" 
+        type="radio"
+        required
+        value="ten_twenty"
+        name="groupSizeValue"
+        class="form-input-radio">
+
+        <label 
+        for="tenTwenty"
+        class="form-label-input">
+        10 - 20</label>
+      </div> -->  
+        
+      <!-- dates: date & checkbox -->
+
+      <h3 class="form-label-heading">Dates</h3>
+
+      <div class="form-dates-group">
+        <label 
+        for="dateArrival"
+        class="form-label-input">
+        Arrival</label>
+
+        <input 
+        id="dateArrival" 
+        type="date"
+        required
+        name="dateArrivalValue"
+        min="2020-12-01"
+        max="2021-04-01"
+        class="form-input-date">
+      </div>
+          
+      <div class="form-dates-group">
+        <label 
+        for="dateDeparture"
+        class="form-label-input">
+        Depature</label>
+
+        <input 
+        id="dateDeparture" 
+        type="date"
+        required
+        name="dateDepartureValue"
+        min="2020-12-01"
+        max="2021-04-01"
+        class="form-input-date">
+      </div>
+
+      <div class="form-dates-checkbox-group">
+        <label 
+        for="flexible"
+        class="form-label-input">
+        My dates are flexible</label>
+
+        <input
+        id="flexible"
+        type="checkbox"
+        name="flexibleDateValue"
+        value="flexible"
+        class="form-input-checkbox">
       </div>
+          
+      <!-- extras features: text -->
+      <div class="form-extra-group">
+          <label 
+          for="otherComments"
+          class="form-label-heading">
+          Extra features</label>
+
+          <input 
+          id="otherComments" 
+          type="text"
+          name="otherCommentsValue"
+          placeholder="Tell us exactly what you want"
+          class="form-input-text">
+        </div>
```

Comment:
Be consistent with the indentation (2 spaces, and closing line as much indented as the opening tag):
```
<div class="form-extra-group">
  <label 
    for="otherComments"
    class="form-label-heading">
Extra features
  </label>
  <input 
    id="otherComments" 
    type="text"
    name="otherCommentsValue"
    placeholder="Tell us exactly what you want"
    class="form-input-text">
</div>

```